iocore/net/UnixNetAccept.cc

```diff
@@ -470,6 +470,7 @@ NetAccept::acceptFastEvent(int event, void *ep)
     } else {
       close_UnixNetVConnection(vc, e->ethread);
     }
+    vc = nullptr;
```

Comment:
If ATS has callbacked or close the vc, the NetAccept must not touch the vc in the next loop.